devicon.json

```
@@ -8155,14 +8155,15 @@
         "versions": {
             "svg": [
                 "original",
-                "plain"
+                "plain",
+                "line"
             ],
             "font": [
-                "original",
-                "plain"
+                "plain",
+                "line"
             ]
         },
-        "color": "#2369bc",
+        "color": "#000",
```

Perhaps we can use the color in-between the two blues in the color-gradient as the main color?
```suggestion
        "color": "#1f65b7",
```

Here's how I calculated the color: https://jsfiddle.net/5p2xjucv/26/